src/features/data-streams/common/gettingStarted.mdx

```diff
@@ -1,16 +1,11 @@
-import { Aside, CodeSample, CopyText, ClickToZoom } from "@components"
+import { CodeSample, CopyText, ClickToZoom } from "@components"
+import DataStreams from "@features/data-streams/common/DataStreams.astro"
 
-This guide shows you how to read data from a Data Stream, validate the answer, and store the answer onchain. This example uses a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. For this example, the log trigger comes from a simple emitter contract. Chainlink Automation then uses `StreamsLookup` to retrieve a signed report from the Data Streams Engine, return the data in a callback, and run the [`performUpkeep` function](/chainlink-automation/reference/automation-interfaces#performupkeep-function-for-log-triggers) on your registered upkeep contract. The `performUpkeep` function calls the `verify` function on the verifier contract.
+This guide shows you how to read data from a Data Streams feed, validate the answer, and store the answer onchain. This example uses the _[Streams Trade](/data-streams#streams-trade-using-data-streams-with-chainlink-automation) implementation_, with a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. For this example, the log trigger comes from a simple emitter contract. Chainlink Automation then uses `StreamsLookup` to retrieve a signed report from the Data Streams Engine, return the data in a callback, and run the [`performUpkeep` function](/chainlink-automation/reference/automation-interfaces#performupkeep-function-for-log-triggers) on your registered upkeep contract. The `performUpkeep` function calls the `verify` function on the verifier contract.
 
-<Aside type="caution" title="Disclaimer">
-  This guide represents an example of using a Chainlink product or service and is provided to help you understand how to
-  interact with Chainlink's systems and services so that you can integrate them into your own. This template is provided
-  "AS IS" and "AS AVAILABLE" without warranties of any kind, has not been audited, and may be missing key checks or
-  error handling to make the usage of the product more clear. Do not use the code in this example in a production
-  environment without completing your own audits and application of best practices. Neither Chainlink Labs, the
-  Chainlink Foundation, nor Chainlink node operators are responsible for unintended outputs that are generated due to
-  errors in code.
-</Aside>
+Note: If you want to learn how to use the _[Streams Direct](/data-streams#streams-direct-using-the-data-engine-with-your-own-bot) implementation_ of Data Streams, refer to the [Fetch and decode reports via a REST API](/data-streams/tutorials/streams-direct-api) guide or the [Stream and decode reports via WebSocket](/data-streams/tutorials/streams-direct-ws) guide.
+
+<DataStreams section="asideDisclaimer" />
 
 ## Before you begin
 
```

Comment:
There's also an Arbitrum Sepolia ETH faucet at Alchemy (https://www.alchemy.com/faucets/arbitrum-sepolia) if you want to include that as an option